project/Dependencies.scala

```diff
@@ -113,4 +113,11 @@ object Dependencies {
 
   val scalaXml = scala211Module("scala-xml", "1.0.6")
   val scalaParsers = scala211Module("scala-parser-combinators", "1.0.5")
+
+  def log4jVersion = "2.8.1"
+  val log4jApi = "org.apache.logging.log4j" % "log4j-api" % log4jVersion
+  val log4jCore = "org.apache.logging.log4j" % "log4j-core" % log4jVersion
+  val log4jSlf4jImpl = "org.apache.logging.log4j" % "log4j-slf4j-impl" % log4jVersion
+  // specify all of log4j modules to prevent misalignment
+  val log4jDependencies = Vector(log4jApi, log4jCore, log4jSlf4jImpl)
```

Comment:
This adds SLF4J binding for Log4j 2. Until now we've had Log4j 2, but not the binding. The scripted test of sbt-slf4j now picks up sbt as the logger, making the plugin obsolete as @larsrh noted.